tools/release/adot-operator-images-mirror/main.go

```diff
@@ -31,6 +31,7 @@ type Repository struct {
 	Registry string `yaml:"registry"`
 	Name     string `yaml:"name"`
 	Host     string `yaml:"host"`
+	TagsAllowlist []string `yaml:"tags_allow_list"`
```

Comment:
```suggestion
	AllowedTags []string `yaml:"allowed_tags"`
```

I prefer this, not sure I can express exactly why, though. v0v

Reply:
Changed